deploy/sql-migrations/migrate-db.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
```

Comment:
Why do we need the same file twice?

Reply:
ah yes, forgot to remove the one in migrations dir!

Reply:
removed